components/button-cards/solution-card/solution-card.js

```diff
@@ -0,0 +1,48 @@
+const cardRows = document.querySelectorAll('.card-row')
+
+calcCardDimension()
+window.onresize = function() {
+  calcCardDimension()
+}
+
+function calcCardDimension() {
+  for (let i = 0; i < cardRows.length; i++) {
+    const cardRow = cardRows[i]
+    const heighestImage = getHeighestImage(cardRow)
+
+    if (screen.width < 992) {
+      setCardRowheight(cardRow, heighestImage)
+    } else {
+      setCardheight(cardRow, heighestImage)
+    }
+  }
+}
+
+function getHeighestImage(cardRow) {
```

Comment:
Aren't all the images going to be the same height, or should we set the same height for all images? I think this code is good for the generic cards that adapt based on large images, but I don't think it should be used for the solution cards.
Maybe chat to Martin and double check